cdap-app-fabric/src/test/java/co/cask/cdap/AppWithMultipleScheduledWorkflows.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap;
+
+import co.cask.cdap.api.app.AbstractApplication;
+import co.cask.cdap.api.workflow.AbstractWorkflow;
+import co.cask.cdap.api.workflow.AbstractWorkflowAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ */
+public class AppWithMultipleScheduledWorkflows extends AbstractApplication {
+
+  @Override
+  public void configure() {
+    setName("AppWithMultipleScheduledWorkflows");
+    setDescription("Sample application with multiple Workflows");
+    addWorkflow(new SomeWorkflow());
+    addWorkflow(new AnotherWorkflow());
+    scheduleWorkflow("SomeSchedule1", "0 4 * * *", "SomeWorkflow");
```

Comment:
Schedules should be set while configuring the adapter. There was a comment about this earlier?